daemons/dss-sync/app.py

```diff
@@ -24,7 +29,93 @@ def process_new_syncable_object(event, context):
     if event.get("Event") == "s3:TestEvent":
         app.log.info("DSS sync daemon received S3 test event")
     else:
-        s3 = boto3.resource("s3")
-        bucket = s3.Bucket(event['Records'][0]["s3"]["bucket"]["name"])
+        bucket = resources.s3.Bucket(event['Records'][0]["s3"]["bucket"]["name"])
         obj = bucket.Object(unquote(event['Records'][0]["s3"]["object"]["key"]))
-        sync_blob(source_platform="s3", source_key=obj.key, dest_platform="gs", logger=app.log)
+        sync_blob(source_platform="s3", source_key=obj.key, dest_platform="gs", logger=app.log, context=context)
+
+@app.sns_topic_subscriber("dss-gs-composite-upload-ready")
+def compose_upload(event, context):
```

Comment:
Added a sample event, as well as an exposition docstring at the top of this file.

Also added a todo for myself to protocolize the SNS messages somehow.